### PR TITLE
Fix docs for CI

### PR DESCRIPTION
**Pull request by pinkforest**
#873 Docs run is [Failing](https://github.com/railwayapp/nixpacks/actions/runs/4963171636/jobs/8882246761?pr=873) unrelated to my change.

This irons out some tsx it complains about - not entirely sure yet about if these work :woman_shrugging: 

There is also rats nest of warnings about missing default export on imports.

### CODE REVIEW

File: docs/components/SideNav.tsx [hunk 1]
```diff
@@ -18,10 +18,8 @@ export const SideNav: React.FC<{ className?: string }> = ({ className }) => {
       {/* using css fallback properties for cross-browser compatibility  */}
       {/* read more: https://modernweb.com/using-css-fallback-properties-for-better-cross-browser-compatibility/ */}
       <style jsx>
-        {" "}
```
Comment (pinkforest) [inline docs/components/SideNav.tsx:21]: I wonder why was this needed resulting two entries the linter is complaining about ? There was no comment.

File: docs/components/SideNav.tsx [hunk 2]
```diff
@@ -18,10 +18,8 @@ export const SideNav: React.FC<{ className?: string }> = ({ className }) => {
...
         {`
           .sidebar-container {
-            height: calc(100vh - var(--top-nav-height) - 4px);
```
Comment (pinkforest) [inline docs/components/SideNav.tsx:24]: I wonder why was this needed resulting two entries the linter is complaining about ? There was no comment.

Comment (Miloas) [reply docs/components/SideNav.tsx:24]: please dont remove this, there is a comment above this. `100vh` is a fallback of `100dvh`

**Comment by Miloas** [comment]
Sorry, I think its my pr https://github.com/railwayapp/nixpacks/pull/883 makes it failed, let me think how to fix it

**Comment by Miloas** [comment]
@coffee-cup hey, what issues did you meet? I found you did this https://github.com/railwayapp/nixpacks/commit/69792d2c37b90828815896d250ab1bcb4fa62904, but after I revert this, `next build` and `yarn lint` works fine

**Comment by pinkforest** [comment]
Thanks - I'll let you people fix this one :)
